toolkit/tools/imagecustomizer/container/build-container.sh

```diff
@@ -52,22 +52,22 @@ function cleanUp() {
 }
 trap 'cleanUp' ERR
 
-micLocalFile="$enlistmentRoot/toolkit/out/tools/imagecustomizer"
-micLicensesDir="$enlistmentRoot/toolkit/out/tools/LICENSES/imagecustomizer"
+LocalFile="$enlistmentRoot/toolkit/out/tools/imagecustomizer"
+LicensesDir="$enlistmentRoot/toolkit/out/tools/LICENSES/imagecustomizer"
 
 stagingBinDir="${containerStagingFolder}/usr/local/bin"
 stagingLicensesDir="${containerStagingFolder}/usr/local/share/licenses"
 
-dockerFile="$scriptDir/Dockerfile.mic-container"
```

Comment:
There should still be some prefix added to the `Dockerfile`. This is a way of signaling that the `Dockerfile` is not usable directly.